components/python/pyzmq/Makefile

```diff
@@ -10,15 +10,32 @@
 
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
+# Copyright 2016 Jim Klimov
 #
 
+# See zeromq configure.ac for ABI version mapping to software version
```

Comment:
As said in the comment and code a few lines lower, this is needed to set up `C*FLAGS` to include and link the desired zeromq version as you asked, if we have the choice of several versions. For that version to have priority over others, these values must be set before we include the oi-userland makefiles and add common include/lib-dirs.

At least this is the intention, which seemed to work for my local build-testing. I am not sure this is the really correct approach to this problem, so am open to suggestions.